tests/chez/chez031/expected

```diff
@@ -0,0 +1 @@
+Uncaught error: Specifiers.idr:18:1--24:5:No recognised foreign calling convention
```

Comment:
Seeing "Uncaught error" is a bit scary. Is there any way to print
something nicer? Would be cool if the compiler could also say something
like "I was hoping to find ... because option ... was passed".

Reply:
That's the normal error you'd get if you wrote a specifier that didn't match any backends, e.g.
```idris
%foreign "fizz:bop,pop"
fizz : Int -> Int
```
since there's no fizz backend.
I'm not clear what you'd prefer to see but I am interested, could you elaborate?

Reply:
When we have a type error we pretty-print an error message with some
location, explanation, etc. we don't just let an "uncaught error"
escape.

Could we do the same here? If the specifier did not match any of the
existing backends, we could for instance list them so that they are
easier to discover.

Reply:
Sure, sounds like a good idea. I'm not sure how specific the error should be but this should give the user some leads and is no longer uncaught.